### PR TITLE
[POWERCFG] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/cpl/powercfg/lang/ro-RO.rc [hunk 1]
```diff
@@ -4,110 +4,110 @@
...
     COMBOBOX IDC_ENERGYLIST, 14, 54, 224, 92, CBS_DROPDOWNLIST | CBS_AUTOHSCROLL | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "Salvea&ză…", IDC_SAVEAS_BTN, 85, 72, 80, 14, WS_DISABLED
-    PUSHBUTTON "Șt&erge", IDC_DELETE_BTN, 173, 72, 64, 14, WS_DISABLED
+    PUSHBUTTON "Sal&vare ca…", IDC_SAVEAS_BTN, 85, 72, 80, 14, WS_DISABLED
+    PUSHBUTTON "Şt&ergere", IDC_DELETE_BTN, 173, 72, 64, 14, WS_DISABLED
     GROUPBOX "", IDC_GRPDETAIL, 6, 95, 240, 119
-    LTEXT "Când calculatorul este:", IDC_SAT, 12, 110, 60, 15
-    LTEXT "În priză", IDC_SAC, 100, 110, 50, 15
+    LTEXT "Când computerul este alimentat:", IDC_SAT, 12, 110, 60, 15
+    LTEXT "De la reţeaua electrică", IDC_SAC, 100, 110, 50, 15
```
Comment (JoachimHenze) [inline dll/cpl/powercfg/lang/ro-RO.rc:25]: This definitely does not fit the 50 width.

File: dll/cpl/powercfg/lang/ro-RO.rc [hunk 2]
```diff
@@ -4,110 +4,110 @@
...
+    GROUPBOX "&Scheme de alimentare", -1, 6, 43, 240, 50
     COMBOBOX IDC_ENERGYLIST, 14, 54, 224, 92, CBS_DROPDOWNLIST | CBS_AUTOHSCROLL | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "Salvea&ză…", IDC_SAVEAS_BTN, 85, 72, 80, 14, WS_DISABLED
-    PUSHBUTTON "Șt&erge", IDC_DELETE_BTN, 173, 72, 64, 14, WS_DISABLED
+    PUSHBUTTON "Sal&vare ca…", IDC_SAVEAS_BTN, 85, 72, 80, 14, WS_DISABLED
+    PUSHBUTTON "Şt&ergere", IDC_DELETE_BTN, 173, 72, 64, 14, WS_DISABLED
     GROUPBOX "", IDC_GRPDETAIL, 6, 95, 240, 119
-    LTEXT "Când calculatorul este:", IDC_SAT, 12, 110, 60, 15
-    LTEXT "În priză", IDC_SAC, 100, 110, 50, 15
+    LTEXT "Când computerul este alimentat:", IDC_SAT, 12, 110, 60, 15
```
Comment (JoachimHenze) [inline dll/cpl/powercfg/lang/ro-RO.rc:24]: Unlikely that this will fit the 60 width

File: dll/cpl/powercfg/lang/ro-RO.rc [hunk 3]
```diff
@@ -4,110 +4,110 @@
...
+    LTEXT "De la reţeaua electrică", IDC_SAC, 100, 110, 50, 15
     ICON IDI_AC, IDC_IAC, 141, 122, 0, 0
-    LTEXT "Pe baterii", IDC_SDC, 173, 110, 50, 15
+    LTEXT "De la baterii", IDC_SDC, 173, 110, 50, 15
```
Comment (JoachimHenze) [inline dll/cpl/powercfg/lang/ro-RO.rc:27]: unlikely that this would fit the 50 width

File: dll/cpl/powercfg/lang/ro-RO.rc [hunk 4]
```diff
@@ -4,110 +4,110 @@
...
+    LTEXT "Selectaţi setările pentru economisirea energiei.", -1, 37, 6, 208, 20
     GROUPBOX "Opțiuni", -1, 6, 30, 240, 65
-    AUTOCHECKBOX "Afișează pictogramă în &zona de notificare", IDC_SYSTRAYBATTERYMETER, 17, 43, 220, 10, WS_TABSTOP
-    AUTOCHECKBOX "Solicită &reautentificare la revenirea din starea de veghe", IDC_PASSWORDLOGON, 17, 60, 220, 10, WS_TABSTOP
-    AUTOCHECKBOX "Înt&unecă ecranul când calculatorul merge pe baterii", IDC_VIDEODIMDISPLAY, 17, 78, 220, 10, WS_TABSTOP
-    GROUPBOX "Consecințe ale acțiunilor de utilizator", -1, 6, 101, 240, 110
-    LTEXT "&Plierea calculatorul portabil determină:", IDC_SLIDCLOSE, 15, 115, 222, 10
+    AUTOCHECKBOX "Afişa&re pictograme pe bara de activităţi", IDC_SYSTRAYBATTERYMETER, 17, 43, 220, 10, WS_TABSTOP
+    AUTOCHECKBOX "&Solicitare parolă când computerul iese din starea de veghe", IDC_PASSWORDLOGON, 17, 60, 220, 10, WS_TABSTOP
+    AUTOCHECKBOX "&Umbrire afişare când se alimentează de la baterii.", IDC_VIDEODIMDISPLAY, 17, 78, 220, 10, WS_TABSTOP
```
Comment (JoachimHenze) [inline dll/cpl/powercfg/lang/ro-RO.rc:104]:
```suggestion
    AUTOCHECKBOX "&Umbrire afişare când se alimentează de la baterii", IDC_VIDEODIMDISPLAY, 17, 78, 220, 10, WS_TABSTOP
```
MS does (almost?) never end texts in AUTOCHECKBOXES with a dot. I am pretty sure that the dot in the end is wrong.

File: dll/cpl/powercfg/lang/ro-RO.rc [hunk 5]
```diff
@@ -117,64 +117,64 @@ CAPTION "Hibernare"
...
+    IDS_TIMEOUT8 "După 25 minute"
+    IDS_TIMEOUT9 "După 30 minute"
+    IDS_TIMEOUT10 "După 45 minute"
+    IDS_TIMEOUT11 "După 1 oră"
+    IDS_TIMEOUT12 "După 2 ore"
+    IDS_TIMEOUT13 "După 3 ore"
+    IDS_TIMEOUT14 "După 4 ore"
+    IDS_TIMEOUT15 "După 5 ore"
+    IDS_TIMEOUT16 "Niciodată"
+    IDS_DEL_SCHEME "Sigur ştergeţi schema de alimentare: ""%s""?"
```
Comment (JoachimHenze) [inline dll/cpl/powercfg/lang/ro-RO.rc:177]:
```suggestion
    IDS_DEL_SCHEME "Sigur ştergeţi schema de alimentare: '%s'?"
```
Just an idea. I am not sure what MS does use here actually. No need to change it if you prefer the other way.

File: dll/cpl/powercfg/lang/ro-RO.rc [hunk 6]
```diff
@@ -4,110 +4,110 @@
...
     ICON IDI_AC, IDC_IAC, 141, 122, 0, 0
-    LTEXT "Pe baterii", IDC_SDC, 173, 110, 50, 15
+    LTEXT "De la baterii", IDC_SDC, 173, 110, 50, 15
     ICON IDI_DC, IDC_IDC, 218, 122, 0, 0
-    LTEXT "Închiderea e&cranului:", IDC_MONITOR, 13, 148, 85, 10
+    LTEXT "Stingere &monitor:", IDC_MONITOR, 13, 148, 85, 10
     COMBOBOX IDC_MONITORACLIST, 100, 145, 65, 150, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     COMBOBOX IDC_MONITORDCLIST, 173, 145, 65, 150, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    LTEXT "Oprirea &discurilor:", IDC_DISK, 13, 163, 85, 10
+    LTEXT "Oprire d&iscuri fixe:", IDC_DISK, 13, 163, 85, 10
```
Comment (JoachimHenze) [inline dll/cpl/powercfg/lang/ro-RO.rc:32]: MS really used an i as accelerator? That is untypically. Please attach a pic from the romanian MS dlg for proof.

File: dll/cpl/powercfg/lang/ro-RO.rc (hunk 5 above)
Comment (AndreiMiloiu) [reply dll/cpl/powercfg/lang/ro-RO.rc]: I was tired yesterday and I didn't pay attention. In every language there is "" sign and not ''. This is the same in Windows, too. Here is a screenshot. I will revert this change.

File: dll/cpl/powercfg/lang/ro-RO.rc (hunk 6 above)
Comment (AndreiMiloiu) [reply dll/cpl/powercfg/lang/ro-RO.rc:32]: Yes, it's i letter an accelerator. Here is the proof.

2024-04-14 11_31_47-Resource Hacker - powercfg cpl mui

Comment (JoachimHenze) [reply dll/cpl/powercfg/lang/ro-RO.rc:32]: Resolved, thanks.

File: dll/cpl/powercfg/lang/ro-RO.rc (hunk 4 above)
Comment (JoachimHenze) [reply dll/cpl/powercfg/lang/ro-RO.rc]: Resolved, thanks.

File: dll/cpl/powercfg/lang/ro-RO.rc (hunk 5 above)
Comment (JoachimHenze) [reply dll/cpl/powercfg/lang/ro-RO.rc:177]: Ok, no problem. Resolved, thanks.

**Review by JoachimHenze — APPROVED** [review]
Approved now.

**Pull request merged**